src/client/styles/fonts/_frutiger.scss

```diff
@@ -0,0 +1,7 @@
+@import url("https://fast.fonts.net/cssapi/0985eb4d-eec4-488b-baa4-22fe85c5ce32.css");
+body p, h1, h2, h3, details summary, ul {
```

Comment:
I've tried with body only but the styling has been overwritten because it's specifically set against these selectors in core front-end node module. (All was set to Arial)
If I use the specific selectors it works.
I've tried now your generic set. Didn't work either.
I didn't want to use the !important.
I am happy to implement any suggestions.

In the node module I can find this:
.local-header__title {
  @include bold-font(36);
  color: $white;
  margin: 0;

  @include media(tablet) {
    padding-top: $baseline-grid-unit * 4;
  }
}
This will overwrite the local header title to Arial.